README.md

```diff
@@ -18,10 +20,18 @@ Prerequisites: A GitHub account (for logging into Gitpod)
 * After opening the tool via the notification, the interface loads in a few seconds.
 * Now you're ready to try the trace viewer!
     * Head to the trace viewer tab in the left side menu to get started.
-    * The tool is already loaded with example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab), so no need to hunt for your own.
+    * The tool is loaded with the example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab). To analyse your own traces, [download the application](https://github.com/theia-ide/theia-trace-extension#download-the-application) for Linux.
 
 ![gitpod-live-demo-setup](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/gitpod-live-demo-setup.gif)
 
+## Download the application 
```

Comment:
This chapter makes it sound that the app image is a deliverable of this github project and part of this repository. You need to make clear in this section that it's an external build and user can download it from another web-page and try it. Change the title and the description below.

Reply:
Which of these points needs to be clarified?
* The downloadable application isn't built or hosted by the official repo maintainers
* The application isn't a direct product of this repo, it also requires Theia (This repo isn't for an application, it's for an extension)

Reply:
How about this? This makes makes it more clear that the application isn't a direct product of the repo, but I'm not sure that's what you wanted.

----

## Download an application
If you'd like to explore your own traces, you can **[download a theia-based IDE with this trace viewer extension here](https://www.dorsal.polymtl.ca/files/other/electron-theia-trace-example-app-0.0.1.AppImage)!**